client/auth/edit-account.jsx

```diff
@@ -50,14 +50,14 @@ const emailValidator = composeValidators(
   minLength(
     EMAIL_MINLENGTH,
     t('common.validators.minLength', {
-      defaultValue: `Use at least {{minLength}} characters`,
+      defaultValue: `Enter at least {{minLength}} characters`,
```

Comment:
Think it'd probably be better to have something that deals with the defaultValue and just takes in the minLength/maxLength/etc. (could even be a hook, `useMinLengthValidator`?). So that we don't need to repeat + fix this default value everywhere (even though I know it will be checked for same-ness).

This can be fixed in a different PR though